core/src/main/java/com/xatkit/core/recognition/nlpjs/adapter/ResolutionDeserializer.java

```diff
@@ -0,0 +1,36 @@
+package com.xatkit.core.recognition.nlpjs.adapter;
```

Comment:
This class needs some documentation: it's not clear why we need it and where it's used.

Reply:
I needed to implement this adapter to have a `value` field of type `Object` for parameter instances and adapt the values returned by the REST API to Object. Check https://github.com/axa-group/nlp.js/blob/master/docs/v3/builtin-entity-extraction.md.
This adapter is called (instead of the default one) when deserializing the extracted entities and their values.